autonomy/deploy/chain.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2022 Valory AG
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+
+"""Utils to support on-chain contract interactions."""
+
+
+import os
+from typing import Dict, Optional
+
+import requests
+import web3
+
+
+SERVICE_REGISTRY_ABI = "https://abi-server.staging.autonolas.tech/autonolas-registries/ServiceRegistry.json"
+DEFAULT_STAGING_CHAIN = "https://chain.staging.autonolas.tech"
+
+CHAIN_CONFIG: Dict[str, Dict[str, Optional[str]]] = {
+    "staging": {
+        "rpc": os.environ.get("STAGING_CHAIN_RPC", DEFAULT_STAGING_CHAIN),
+        "address": os.environ.get(
+            "SERVICE_ADDRESS_STAGING", "0x0DCd1Bf9A1b36cE34237eEaFef220932846BCD82"
+        ),
+    },
+    "ethereum": {
+        "rpc": os.environ.get("ETHEREUM_CHAIN_RPC"),
+        "address": os.environ.get(
+            "SERVICE_ADDRESS_ETHEREUM", "0x48b6af7B12C71f09e2fC8aF4855De4Ff54e775cA"
+        ),
+    },
+    "goerli": {
+        "rpc": os.environ.get("GOERLI_CHAIN_RPC"),
+        "address": os.environ.get(
```

Comment:
```suggestion
        "service_address": os.environ.get(
```
otherwise this is too ambiguous